namespaces.go

```diff
@@ -41,12 +41,11 @@ type Namespace struct {
 // CreateNamespace creates a new Namespace. Returns error if could not be created
 // or another namespace already exists.
 func (h *Headscale) CreateNamespace(name string) (*Namespace, error) {
-	var err error
-	namespace := Namespace{}
-	name, err = NormalizeNamespaceName(name)
+	err := CheckNamespaceName(name)
```

Comment:
I agree on the server side, and I think it should be checked in this function, and rename. But not checked in Get and Destroy. Get and Destroy should return a "namespace not found" not invalid.

Reply:
Good point, I'll make that change